carma-messenger-core/traffic_incident/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(traffic_incident)
+
+## Compile as C++14, supported in ROS Kinetic and newer
+add_compile_options(-std=c++14)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  carma_utils
+  cav_msgs
+  gps_common
+  roscpp
+)
+
+
+###################################
+## catkin specific configuration ##
+###################################
+## The catkin_package macro generates cmake config files for your package
+## Declare things to be passed to dependent projects
+## INCLUDE_DIRS: uncomment this if your package contains header files
+## LIBRARIES: libraries you create in this project that dependent projects also need
+## CATKIN_DEPENDS: catkin_packages dependent projects also need
+## DEPENDS: system dependencies of this project that dependent projects also need
+catkin_package(
+   INCLUDE_DIRS include
+#  LIBRARIES traffic_incident
+   CATKIN_DEPENDS carma_utils cav_msgs gps_common roscpp
+#  DEPENDS system_lib
+)
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
+## Your package locations should be listed before other locations
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+add_executable(traffic_incident_node src/main.cpp src/traffic_incident_node.cpp src/traffic_incident_worker.cpp)
```

Comment:
let's remove src/traffic_incident_worker.cpp from here as it is included in the traffic_incident_worker_lib.
No reference of file (.cpp tpp etc) should be duplicated in the CMake file :D